.github/workflows/helm-release.yaml

```diff
@@ -46,8 +47,8 @@ jobs:
           curl -L "https://github.com/helm/chart-releaser/releases/download/v${CR_VERSION}/chart-releaser_${CR_VERSION}_linux_amd64.tar.gz" | sudo tar xvz --directory /usr/bin cr
 
           make dev/tools
-          git checkout go.*
       - name: Update chart versions
+        id: update-chart-version
```

Comment:
not used anywhere but I guess it does not hurt

Reply:
It's actually used in my matching PR in KM